app/views/admin/invites/index.html.haml

```diff
@@ -0,0 +1,42 @@
+.row
+  .col-md-12
+    .page-header
+      %h1
+        Invitations
+
+.row
+  .col-md-12
+    - if @invites.present?
+      %table.datatable
+        %thead
+          %th
+            %b ID
+          %th
+            %b User
+          %th
+            %b Invited for
+          %th
+            %b Invitation validity
+          %th
+            %b Action
+
+        - @invites.each do |invite|
+          %tr
+            %td
+              = invite.id
+            %td
+              = invite.emails
+            %td
+              = invite.invite_for
+            %td
+              = invite.end_date
+            %td
+              = link_to 'Edit', edit_admin_conference_invite_path(@conference.short_title, invite.id),
+              class: 'btn btn-primary'
+              = link_to 'Delete', admin_conference_invite_path(@conference.short_title, invite.id), method: :delete,
```

Comment:
There should be a confirmation dialog for deletion to avoid mistakes

Reply:
ok!